azure-pipelines.yml

```diff
@@ -1,6 +1,6 @@
 ---
 trigger:
-  - master
```

Comment:
Why trigger set to none...Please change it back to master....

Reply:
changed to master